src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObj = {};
+  const makeArray = sourceString.split(';');
+  const stylesArray = makeArray.map(element => element.trim().split(':'))
+    .filter(arr => arr.length > 1);
+
+  for (const style of stylesArray) {
```

Comment:
[checklist - 5](https://github.com/mate-academy/js_style-to-object/blob/master/checklist.md)

Reply:
maybe you could use `reduce` method here?